PsfLauncher/main.cpp

```diff
@@ -304,7 +315,87 @@ void LaunchMonitorInBackground(std::filesystem::path packageRoot, const wchar_t
         }
     }
 }
+
+void LogTelemetryData()

```

Comment:
Function name is not too clear on what we are trying to log here.

Reply:
what about ReadApplicationAndProcessesCollection?